cirq-web/cirq_ts/dist/index.html

```diff
@@ -4,11 +4,16 @@
   <title>Cirq Web Development page</title>
 </head>
 <body>
-  <div id="container"></div>
-  <script src="/bloch_sphere.bundle.js"></script>
+  <div id="mycircuitdiv"></div>
```

Comment:
This was my point in the previous PR - how are you going to have now a dev environment for the bloch sphere? 
My recommendation would be to have this as a top level HTML file, which refers to two sub HTML files like a little "table of contents": 

```
Welcome to the Cirq Web interactive development page! 

Available dev pages: 
* 3D Bloch Sphere 
* 3D Circuit
```

And then each would have their own little page.